common/opinion_modifiers/hfe_opinion.txt

```diff
@@ -22,7 +22,7 @@ hfe_opinion_egypt_vs_varelse = {
 		}
 	}
 	opinion = {
-		base = -100
+		base = -999
```

Comment:
Too much

Reply:
Bull crap! The Ju Ahets nearly committed total genocide on the entire Varelsite race, down to the last reptile! A negative one hundred Opinion CAN BE OVERCOME. Could you imagine if the Forsaken Ones had an opinion like this?

Did the Jews ever befriend the Nazis? NO!